packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts

```diff
@@ -71,6 +72,7 @@ const chainMetadataByName: Record<ChainName, ChainMetadata | null> = {
   mainnet: { chainId: 1, ...ethereumIcon },
   optimism: { chainId: 10, ...optimismIcon },
   optimismKovan: { chainId: 69, ...optimismIcon },
+  optimismGoerli: { chainId: 420, ...optimismIcon },
```

Comment:
Chain ID looks right to me: https://community.optimism.io/docs/useful-tools/networks/#optimism-goerli